### PR TITLE
chore: improve logs in modify-data-object

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
chore: improve logs in modify-data-object

this commit adds functionality, which will print out, when task is waiting for import to be finished, or when waiting until object is deleted.

**Release note**:
```
NONE
```

### CODE REVIEW

**Comment by jcanocan** [comment]
Thanks @ksimon1!
/lgtm

File: modules/modify-data-object/pkg/dataobject/provider.go [hunk 1]
```diff
@@ -72,6 +72,7 @@ func (d *dataObjectProvider) DeletePVC(namespace string, name string) error {
 }
 
 func waitUntilDataObjectIsDeleted(helper *resource.Helper, namespace, name string) {
+	log.Logger().Info("waiting until Data object is deleted")
```
Comment (0xFelix) [inline modules/modify-data-object/pkg/dataobject/provider.go:75]: nit: Use same case in all messages? `data object` or `Data object`

Comment (ksimon1) [reply modules/modify-data-object/pkg/dataobject/provider.go]: fixed

**Review by 0xFelix — APPROVED** [review]
/lgtm

Thanks!

**Pull request merged**